### ISSUE TITLE
the loss function of relation extraction

### ISSUE THREAD

**Issue by GeneZC** [WindChimeRan/pytorch_multi_head_selection_re#14]
Why using binary cross entropy instead of cross entropy which is more convenient ?

**Comment by GeneZC** [comment]
to address relation overlap
